polygon/astrid/block_consumer.go

```diff
@@ -0,0 +1,81 @@
+package astrid
+
+import (
+	"context"
+	"errors"
+	"runtime"
+	"syscall"
+	"time"
+
+	"github.com/ledgerwatch/log/v3"
+
+	"github.com/ledgerwatch/erigon-lib/chain"
+	"github.com/ledgerwatch/erigon-lib/direct"
+	executionclient "github.com/ledgerwatch/erigon/cl/phase1/execution_client"
+	"github.com/ledgerwatch/erigon/cmd/utils"
+	"github.com/ledgerwatch/erigon/core/types"
+	"github.com/ledgerwatch/erigon/polygon/sync"
+)
+
+type BlockConsumerDependencies struct {
```

Comment:
This looks like a helper to run `sync.Service`. I wonder if this is a temporary thing or not? I'd like to avoid having this extra wrapper. Is it possible to run `sync.Service` directly and move the `terminateProcess` into the devnet package?